accessibility-checker-engine/src/v2/checker/accessibility/nls/index.ts

```diff
@@ -886,15 +886,15 @@ let a11yNls = {
     },
     // JCH - DONE
     "IBMA_Focus_Tabbable": {
-        0: "Component must have at least one tabbable element",
+        0: "Component must have at least one tabable element",
         "Pass_0": "Rule Passed",
-        "Potential_1": "Component with \"{0}\" role does not have a tabbable element"
+        "Potential_1": "Component with \"{0}\" role does not have a tabable element"
     },
     // JCH - DONE
     "IBMA_Focus_MultiTab": {
-        0: "Certain components must have no more than one tabbable element",
+        0: "Certain components must have no more than one tabable element",
         "Pass_0": "Rule Passed",
-        "Potential_1": "Component with \"{0}\" role has more than one tabbable element"
+        "Potential_1": "Component with \"{0}\" role has more than one tabable element"
```

Comment:
Tabbable was correct

Reply:
agree. This one shoud not have been changed. correcting back